src/Control/Effect/Pure.hs

```diff
@@ -8,18 +8,11 @@ module Control.Effect.Pure
 import Control.Applicative
 import Control.Effect.Carrier
 import Data.Coerce
+import GHC.Generics (Generic1)
```

Comment:
Do we want to export `Generic1` from `Control.Effect.Sum` or something?

Reply:
I don’t think so. It’s only needed about half the time, and it’s in `base`.